client-sdk/go/modules/roflmarket/roflmarket.go

```diff
@@ -303,6 +304,11 @@ func NewInstanceCreateTx(fee *types.Fee, body *InstanceCreate) *types.Transactio
 	return types.NewTransaction(fee, methodInstanceCreate, body)
 }
 
+// NewInstanceTransferAdmin generates a new roflmarket.InstanceTransferAdmin transaction.
```

Comment:
How can one figure out which methods must be called by admin?  Should that be in the comments?

Reply:
Added.